srcpkgs/ocs-url/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'ocs-url'
+pkgname=ocs-url
+version=3.1.0
+revision=1
+#archs="x86_64"
+wrksrc=$pkgname-release-$version
+build_style=qmake
+hostmakedepends="qt5-qmake xdg-utils git"
+makedepends="qt5-devel qt5-svg-devel qt5-declarative-devel qt5-quickcontrols2-devel"
+short_desc="Install helper program for items served via (ocs://)"
+maintainer="Guilherme Machado <machadofguilherme@gmail.com>"
+license="GPL-3.0"
+homepage="https://www.opencode.net/akiraohgaki/$pkgname"
+distfiles="https://www.opencode.net/akiraohgaki/ocs-url/-/archive/release-$version/$pkgname-release-$version.tar.gz"
+checksum=d8c67920359c36eb0004221e5b5695fd04cbc9e93811c372837bdb2a8c9cbcc8
+
+pre_configure() {
+./scripts/prepare
+}
+
+do_configure() {
+qmake PREFIX="/usr"
```

Comment:
Remove do_configure block, it is not need.

Reply:
> Remove do_configure block, it is not need.

Removed. =]